spec/assert.js

```diff
@@ -5,9 +5,9 @@ var assert = {
       return true;
     }
     else {
-      throw new Error("Expected " + assert + " to equal " + expect + ".");
+      console.log("Expected " + assert + " to equal " + expect + ".");
```

Comment:
Can we add the following here:
- a return false
- console.log changed to include "Test failed: " at the start for clarity